doc/sphinx/source/recipes/recipe_sea_surface_salinity.rst

```diff
@@ -9,8 +9,8 @@ Overview
 This recipe compares the regional means of sea surface salinity with a
 reference dataset (ESACCI-SEA-SURFACE-SALINITY v1 or v2 by default).
 To do this, the recipe generates plots for the timeseries of each region and
-a radar plot showing the correlation between dataset and reference timeseries for
-each region during the time they both exist.
+a radar plot showing the bias and standard deviation ratio between dataset 
```

Comment:
change lines 12 and 13 for "radar plots showing (i) the mean state bias, and (ii) the ratio between the simulated and observed standard deviations of different regional averages of sea surface salinity, calculated in the temporal window for which observations and simulations overlap"

Reply:
Added